JsonPatch/PatchOperation.cs

```diff
@@ -161,9 +161,12 @@ internal class Model
 		public JsonElement Value { get; set; }
 	}
 
+	[JsonSerializable(typeof(Model))]
+	internal partial class InternalPatchOperationContext : JsonSerializerContext;
+
 	public override PatchOperation Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 	{
-		var model = JsonSerializer.Deserialize(ref reader, PatchSerializerContext.Default.Model)!;
+		var model = JsonSerializer.Deserialize(ref reader, InternalPatchOperationContext.Default.Model)!;
 
```

Comment:
should these use options.Read? If they were using JsonSerializer.Deserialize in the master branch then it's probably fine to keep.